javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,10 @@
+
+const weekday = ['Sunday', 'Monday', 'Tuseday', 'Wendsday', 'Thursday','Friday','Saturday']
```

Comment:
Nice detail by declaring the weekdays outside the function, because the weekdays will always be the same no matter the `number`, so there is no need to redeclare an array with weekdays on every execution of `getEventWeekday`. :+1: